### PR TITLE
Runtime interaction example

### PR DESCRIPTION
**Pull request by pmikolajczyk41**
Another example project, showing how one can interact with the runtime in drink.

### CODE REVIEW

File: examples/runtime-interaction/README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+This way we can leverage `drink!` to a library for a full-scope runtime simulation engine.
+
+## Running
+
+```bash
+cargo test --release
+```
+
+## `drink::Sandbox` vs `drink::Session`
```
Comment (UladAleph) [inline examples/runtime-interaction/README.md:13]: Could you please elaborate a bit more on when users might choose one over the other. What are the key differences?

File: examples/runtime-interaction/lib.rs [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+        let call_object = RuntimeCall::Contracts(pallet_contracts::Call::remove_code {
+            code_hash: upload_result.code_hash,
+        });
+
+        sandbox
+            .runtime_call(call_object, Some(MinimalRuntime::default_actor()))
+            .expect("Failed to remove a contract");
+    }
+
+    const CONTRACT: &str = r#"
```
Comment (UladAleph) [inline examples/runtime-interaction/lib.rs:69]: A bit bizarre I think. It is not clear for me from the example where this is being taken from. Do you think adding docs to show a way to get this from a contract could be good for the example?

Comment (pmikolajczyk41) [reply examples/runtime-interaction/lib.rs:69]: added some explanation

File: examples/runtime-interaction/README.md (hunk 1 above)
Comment (pmikolajczyk41) [reply examples/runtime-interaction/README.md:13]: added some explanation

**Review by UladAleph — APPROVED** [review]
Solid work :)

File: examples/runtime-interaction/README.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
+# Runtime interaction
+
+This example shows how we can easily send transactions to a blockchain, just like with `subxt` or a similar client.
+
+This way we can leverage `drink!` to a library for a full-scope runtime simulation engine.
```
Comment (piotrMocz) [inline examples/runtime-interaction/README.md]: leverage drink as a library?
I'm not sure but the above doesn't sound quite right to me :D

**Review by piotrMocz — APPROVED** [review]
Awesome :)

File: examples/runtime-interaction/README.md (hunk 3 above)
Comment (pmikolajczyk41) [reply examples/runtime-interaction/README.md]: good point: changed to `leverage 'drink!' to a full-scope runtime simulation engine.`

**Pull request merged**